components/welcome/index.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+import { useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+
+import { TextSampleIcon } from '@/public/icons';
+import SMClickAnimation from '../click-animation';
+
+const texts = [
+  'Once your PR is merged, the token list will update automatically to include your token and your token will be available on the Base Bridge.',
+  'To complete the migration, submit a pull request to the Superchain token repo list.',
```

Comment:
To complete the migration, make sure you verify the migration by adding a link to the verification tweet on the pull request

Reply:
Done